ivy/functional/frontends/tensorflow/__init__.py

```diff
@@ -7,8 +7,8 @@
 from .linalg import *
 from . import math
 from .math import *
-from . import metrics
-from .metrics import *
+from .keras import metrics
```

Comment:
Can you clarify why have you changed the import to `.keras`?

Reply:
The `metrics` module is available under `tf.keras`, so the imports were changed automatically because of the changes I made to the directory structure.